app/assets/stylesheets/registration.scss

```diff
@@ -1,54 +1,79 @@
 .page-registrations {
-  .ball {
-    background: image-url('branding/ball.png') no-repeat;
-    background-size: contain;
-    height: 633px;
-    max-width: 100%;
+  // For the mobile version which doesn't have the same global CSS
+  background: $body-bg;
+
+  #app {
+    margin: -10px;
```

Comment:
On mobile, there is a very small horizontal scrollbar, and I think it has something to do with this, since this makes the page actually 10px bigger (since it increases the size of `#app`), so what you want to do instead is probably reduce the `padding` of `#main` and leave `#app` as it is.

Reply:
Ah nice catch, this is a regression on the last change I made. Fixed, thanks for noticing it.